manifests/services/apache.pp

```diff
@@ -1,13 +1,21 @@
 # This class installs and configures a simple apache webserver, and configures a
 # vhost for the fqdn of the host
 class profile::services::apache {
-  $management_if = hiera('profile::interfaces::management')
-  $default_docroot = hiera('profile::apache::vhost::default::docroot',
-      "/var/www/${::fqdn}")
+  $management_if = lookup('profile::interfaces::management', String)
+  $default_docroot = lookup('profile::apache::vhost::default::docroot', {
+    'value_type'    => Stdlib::Absolutepath,
+    'default_value' => "/var/www/${::fqdn}"
+    })
```

Comment:
Indentering: }) bør kanskje flyttes to hakk til venstre? På alle 3 tilfellene i denne fila.